doc/maintaining/installing/postgres.rst

```diff
@@ -1,5 +1,11 @@
 :orphan:
 
+.. note::
+
+    The following commands should be executed only after running the postgres server. To run 
+    postgres server, execute the command- ``service postgresql start``. 
+    
```

Comment:
Isn't PostgreSQL daemon started automatically on Ubuntu, right after installation? (but, explicit mention won't harm, so i don't have anything particular against this note, just curious)

Reply:
PS so mentioning the necessity of starting PostgreSQL should be in the form of 'if you are facing a problem because of Postgresql is not running, you have to..`. And, unless the user uses root account by default, you also should put `sudo` in the example, just for accordance with other commands. And this hyphen before the command can be omitted, right?

Reply:
Alright, I'll incorporate the relevant change.